e2e/wdio/core/pages/message-toast.po.ts

```diff
@@ -0,0 +1,30 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class MessageToastPo extends CoreBaseComponentPo {
+    private url = '/message-toast';
+
+    section = 'fd-message-toast-example ';
```

Comment:
let's make all the name descriptive. 'section' is understood by you and me, but maybe not to someone else 6 months from now. let's try: exampleBlock or something like that